packages/@okta/vuepress-site/docs/reference/rl-previous/index.md

```diff
@@ -13,22 +13,22 @@ This page covers the rate limits for orgs that were created before 2018-05-17.
 
 If your needs exceed Okta's rate limits, you can purchase the High Capacity Rate Limit add-on. Customers who purchase the High Capacity Rate Limit add-on service may not use the service in excess of the static rate limit, as set forth in the table below. If Okta makes any change to the rate limit, the change is communicated to customers through an updated version of this product documentation.
 
-The following are the high capacity rate limits per minute that apply across the Okta API for these endpoints:
+The following tables lists the high capacity rate limits per minute that apply across the Okta API for these endpoints. You can confirm endpoint limits using the rate limit dashboard in the Admin Console. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
 
 | Endpoint                                                                   | One App   | Enterprise   |
 | -------------------------------------------------------------------------- | ---------: | ------------: |
 | `/oauth2/${authorizationServerId}/v1`                                       | 6000      | 6000         |
 | `/oauth2/v1` except `/oauth2/v1/clients`                                   | 6000      | 6000         |
 | `/api/v1`                                                                  | 3000      | 3000         |
 | `/api/v1/sessions`                                                         | 3000      | 3000         |
-| `/app/template_saml_2_0/${key}/sso/saml`                                    | 3000      | 3000         |
-| `/app/${app}/${key}/sso/saml`                                                | 3000      | 3000         |
-| `/api/v1/groups/${id}`                                                      | 3000      | 3000         |
-| `/api/v1/users/${id}`                                                       | 3000      | 3000         |
+|  Only `/app/template_saml_2_0/${key}/sso/saml`                                    | 3000      | 3000         |
```

Comment:
Maybe use "Exactly" or "Specifically"?

Reply:
As per slack convo, will stick with "Only" for now.